### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by fatima-najafi**
In this branch, I did:

- [ ] Create a popup.js file to implement the popup.
- [ ] edit index.html file.
- [ ] edit styles.css file.

### CODE REVIEW

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @fatima-najafi and @noelfoka , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Used JavaScript to manipulate DOM elements 👍 
Used JavaScript events 👍 
Used objects to store and access data 👍 


## Required Changes ♻️
Please take note that the projects section supposed to be created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). **Hint: This requires changes in existing code - already created HTML code for the projects list should be removed, and instead it should be added via JavaScript.**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/main.js [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+  createModal();
+
+  const buttonsProject = document.querySelectorAll('.works__card button');
+  const closeIcon = document.querySelector('.closeModal');
+  const modal = document.querySelector('.modal');
+  const backgroundModal = document.querySelector('.overlay');
+  const projectName = document.querySelector('.project-name');
+  const projectFeature = document.querySelector('.project-feature');
+  const projectBack = document.querySelector('.project-background');
+  const projectYear = document.querySelector('.project-year');
```
[the remark this answers is not in the document]
Comment (noelfoka) [reply src/main.js:35]: I don't think I can do better based on what I've learned. If you can tell me exactly what to do I would really appreciate it. And also for the close button, it depends on the position of the inspection: when the inspection pallet is at the bottom, you can't see the close button but when it's on the side, the close button can be seen Alright. I would like to know how to solve this problem

[the remark this answers is not in the document]
Comment (OyePriscilla) [reply src/main.js:35]: @noelfoka Foka, actually if you open your pop-p modal; the close button (all the elements in the upper part) are not showing.

[the remark this answers is not in the document]
Comment (noelfoka) [reply src/main.js:35]: please can we have a call please

File: src/main.js [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+  buttonsProject.forEach((button) => {
+    button.addEventListener('click', () => {
+      if (button.id === '0') {
+        projectName.textContent = `${projects[button.id].name}`;
+        projectFeature.textContent = `${projects[button.id].feature}`;
+        projectBack.textContent = `${projects[button.id].background}`;
+        projectYear.textContent = `${projects[button.id].year}`;
+        projectImage.setAttribute('src', `${projects[button.id].image}`);
+        projectDescription.textContent = `${projects[button.id].description}`;
```
Comment (joskalenda) [inline src/main.js:57]: - [ ] I think I will go with the approach suggested by the previous reviewer, it is easy and straightforward to use a `for loop`, iterate over the projects array, and render the `project[i].type`, please make sure you correct the code as for now it is not clean, remember, we are all learner if you are not informed of anything, it is our must to help you use the correct approach.

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @noelfoka , |
|Great work on making the changes requested by a previous reviewer 👏, You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet. |
|  |

## Highlights

- [ ].. ✅ Fixed design
- [ ].. ✅ Green linters
- [ ].. ✅ Great readme file 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Strangeal — APPROVED** [review]
Hi @fatima-najafi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**